scripts/trilinos-integration/README.md

```diff
@@ -0,0 +1,59 @@
+![Kokkos](https://avatars2.githubusercontent.com/u/10199860?s=200&v=4)
+
+# Kokkos: Promotion Test Debugging
+
+This explains the use (and basic implementation details) of promotion testing
+new Kokkos and Kokkos Kernels branches. We first introduce a test for first
+validating a "clean" Trilinos branch that all tests should be passing on.
+Once validated on a clean promotion, we show how to test the same Trilinos branch
+with an updated Kokkoks and Kokkos Kernels.
+
+# Clean Promotion Test
+There is a script called `clean_promotion_test` that tests a self-contained
```

Comment:
Say the script can be found in `scripts/trilinos-integration/`.  My first reaction was where can I find it.

Reply:
Done.